### PR TITLE
Add comments to the main_windows.gd

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: main_window.gd [hunk 1]
```diff
@@ -54,38 +55,45 @@ func parse_logfile():
...
 
 
 func reload_logfile():
-	return open_logfile(current_file_path)
+	return open_logfile(current_file_path) #Returns the function to open the log file in the current file path
 
 
-func scroll_to_last_line():
+func scroll_to_last_line(): 
+#Adds a scrollbar to the logview window that can be moved
```
Comment (Crystalwarrior) [inline main_window.gd]: It gets the already existing scrollbar

File: main_window.gd [hunk 2]
```diff
@@ -54,38 +55,45 @@ func parse_logfile():
...
-	last_date_modified = FileAccess.get_modified_time(path)
-	return current_file
+	$LogfileLabel.text = "Current logfile: " + current_file_path #Add the path to the log file to the LogfileLabel text component 
+	last_date_modified = FileAccess.get_modified_time(path) #Get the current time of when the file on the path was last modified and put that value in the last_time_modified variable
+	return current_file #Return the variable containing the current value of the current_file
 
 
 func reload_logfile():
-	return open_logfile(current_file_path)
+	return open_logfile(current_file_path) #Returns the function to open the log file in the current file path
```
Comment (Crystalwarrior) [inline main_window.gd]: It returns the result of the function

**Comment by ghost** [comment]
Fixed the merge conflict

**Review by Crystalwarrior — CHANGES_REQUESTED** [review]
Better change this to "selected folder"

File: main_window.gd (hunk 1 above)
Comment (ghost) [reply main_window.gd]: Made it more clear

File: main_window.gd (hunk 2 above)
Comment (ghost) [reply main_window.gd]: Rewrote the comment to mention that

**Comment by ghost** [comment]
> Better change this to "selected folder"

Change what to selected folder?

**Comment by Crystalwarrior** [comment]
> > Better change this to "selected folder"
> 
> Change what to selected folder?

Oh oops. Meant the changes in main_window.tscn

**Pull request closed without merging**